client/views/admin/admin.html

```diff
@@ -65,10 +65,11 @@
 												<div>
 													<label><input type="radio" name="{{_id}}" value="1" checked="{{$eq value true}}" /> {{_ "True"}}</label>
 													<label><input type="radio" name="{{_id}}" value="0" checked="{{$eq value false}}" /> {{_ "False"}}</label>
-												</div>
 												{{#if description}}
-													<small>{{description}}</small>
+												<br /><small>{{description}}</small>
 												{{/if}}
+												</div>
+												
```

Comment:
This makes it fall inline a bit better with the other description fields. Before this change:

![descriptionbefore](https://cloud.githubusercontent.com/assets/14922332/10859232/a53b9ba6-7f53-11e5-8569-0cdb7d571968.PNG)

After this change:
![descriptionafter](https://cloud.githubusercontent.com/assets/14922332/10859243/bb1d6ac6-7f53-11e5-8f70-8a47c432d6c8.PNG)